src/content-helper/dashboard-page/pages/traffic-boost/preview/preview-post.php

```diff
@@ -13,8 +13,13 @@
 declare(strict_types=1);
 
 // Define constants to prevent unwanted scripts and styles from being loaded.
-define( 'IFRAME_REQUEST', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
-define( 'WP_ADMIN', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
+
+if ( ! defined( 'IFRAME_REQUEST' ) ) {
+	define( 'IFRAME_REQUEST', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
+}
+if ( ! defined( 'WP_ADMIN' ) ) {
+	define( 'WP_ADMIN', true ); // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedConstantFound
```

Comment:
This is technically an unrelated change in here. I noticed when looking through error logs that the preview-post page load was throwing a warning because `WP_ADMIN` was already defined. I'm not sure what the original purpose here was, but because it's only possible to `define()` once I added these extra guards to avoid the warning.